scripts/release/repackage-dmg.sh

```
@@ -19,10 +28,56 @@ if [ -z "$DMG_PATH" ]; then
+# 2. Resize and replace the app
+# Increase size to ensure the notarized bundle fits (may be slightly larger due to stapling)
+hdiutil resize -size 1g "$DMG_RW"
+
+# Mount the writable image using a temporary directory
+MOUNT_DIR="$(mktemp -d /tmp/OpenHuman-Rebuild-XXXXXX)"
+hdiutil attach "$DMG_RW" -mountpoint "$MOUNT_DIR" -noautoopen
+
+# Replace the non-notarized app with the notarized one
+# We use ditto to preserve all metadata and handles the .app bundle correctly
+APP_NAME="$(basename "$APP_PATH")"
+rm -rf "$MOUNT_DIR/$APP_NAME"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)